test/groovy/templates/PiperPipelineStageReleaseTest.groovy

```diff
@@ -81,7 +86,7 @@ class PiperPipelineStageReleaseTest extends BasePiperTest {
             script: nullScript,
             juStabUtils: utils
         )
-        assertThat(stepsCalled, not(anyOf(hasItem('cloudFoundryDeploy'), hasItem('neoDeploy'), hasItem('kubernetesDeploy'), hasItem('healthExecuteCheck'), hasItem('githubPublishRelease'))))
+        assertThat(stepsCalled, not(anyOf(hasItem('cloudFoundryDeploy'), hasItem('neoDeploy'), hasItem('kubernetesDeploy'), hasItem('healthExecuteCheck'), hasItem('githubPublishRelease'), hasItem('transportRequestUploadCTS'))))
```

Comment:
why transportRequestUploadCTS is called here? config.transportRequestUploadCTS set per default?

Reply:
If `transportRequestUploadCTS: false`, this step would not be called. By default, it is not called.